lib/generators/koi/admin_controller/templates/controller_spec.rb.tt

```diff
@@ -131,5 +131,10 @@ RSpec.describe <%= controller_class_name %>Controller do
       action
       expect(response).to redirect_to(polymorphic_path([:admin, <%= class_name %>]))
     end
+
+    it "deletes the <%= singular_name %>" do
+      model # ensure instance is instantiated
+      expect { action }.to change(<%= class_name %>, :count).by(-1)
```

Comment:
I don't like the implicit ordering approach of referencing model to ensure it exists before the action.

I'm also tending away from using generic names for resources (i.e. model) and using the generator helpers more. This is the spec I wrote for Fringe recently. What do you think of this as an alternative?

```
describe "DELETE /admin/countdowns/:id" do
    let(:action) { delete admin_countdown_path(countdown) }
    let!(:countdown) { create(:countdown) }

    it_behaves_like "requires admin"

    it { is_expected.to redirect_to(admin_countdowns_path) }
    it { expect { action }.to change(Countdown, :count).by(-1) }
  end
  ```

Reply:
I had considered using the explicit `let!` syntax here. I used the approach to keep it consistent with the existing specs, only, so I'm more than happy to make that change.

Also happy to make a change to use named resources rather than `model` as part of this.